packages/twenty-front/src/modules/ui/navigation/action-bar/components/ActionBarItem.tsx

```diff
@@ -80,13 +80,16 @@ export const ActionBarItem = ({ item }: ActionBarItemProps) => {
           }
         />
       ) : (
-        <StyledButton
-          accent={item.accent ?? 'default'}
-          onClick={() => item.onClick?.()}
-        >
-          {item.Icon && <item.Icon size={theme.icon.size.md} />}
-          <StyledButtonLabel>{item.label}</StyledButtonLabel>
-        </StyledButton>
+        <>
+          <StyledButton
+            accent={item.accent ?? 'default'}
+            onClick={() => item.onClick?.()}
+          >
+            {item.Icon && <item.Icon size={theme.icon.size.md} />}
+            <StyledButtonLabel>{item.label}</StyledButtonLabel>
+          </StyledButton>
+          {item.ConfirmationModal}
```

Comment:
Now this needs to be fixed you can either wrap it in a div but its giving error as the modal is relative to actionbaritem component. I hope you are getting the issue. Can you please fix this? If you need any help please ping

Reply:
Hi @Anchit1909 
For the numbering, I think its weird that the width is changing due to change in number of selected rows.

@Bonapara has made some changes in the design 

https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=13736-50538&mode=design&t=xKOKPr1F1nQcJSIR-11

Reply:
Hi @Kanav-Arora! I tried Figma, and I believe keeping the ":" makes it clearer that the actions on the left are the ones targeted by the selection :)

Reply:
Thanks Thomas

@Anchit1909 please proceed with the implementation of figma design

Reply:
Hey @Kanav-Arora , 
I have implemented the new design. 
Also, I tried multiple ways to center the ActionBar without separating the ConfirmationModal, but nothing worked. If I don't separate the ConfirmationModal, it attaches itself to the actionbar and does not take the whole screen as it is absolutely positioned.

Please let me know if this approach is correct or if there is something else that I should do.